src/libos/src/fs/channel.rs

```diff
@@ -43,7 +45,19 @@ impl<I> Channel<I> {
             None,
         );
 
-        Ok(Self { producer, consumer })
+        Ok(Self {
+            producer,
+            consumer,
+            uid,
+        })
+    }
+
+    // Use a random number for uid
+    fn gen_uid(ring_buf: &RingBuffer<I>) -> usize {
+        use crate::util::random;
+        let mut random_buf: [u8; 8] = [0u8; 8]; // same length as usize
+        random::get_random(&mut random_buf).expect("failed to get random number");
```

Comment:
Random does not guarantee two channels would get different id. 
It maybe a good enough implementation for debug purpose. Please add a TODO here, to replace it with a more consistent implementation.

Reply:
In case you are interested, there is an [object_id](https://github.com/occlum/ngo/blob/master/src/libos/crates/object-id/src/lib.rs) crate in NGO for exact this purpose.

Reply:
@tatetian Thank you for this suggestion. However, besides the uniqueness of this uid, it would be better if the developer can find all the references with this ID, which can be very helpful when we need to find the pattern of this access. So the uid should contain many digits. Maybe we can make `object_id` start from a big random value and then atomically increase by 1 or a small random value?